spec/policies/user_policy_spec.rb

```diff
@@ -33,6 +33,9 @@
     it { is_expected.not_to permit_action(:update) }
     it { is_expected.not_to permit_action(:edit) }
     it { is_expected.not_to permit_action(:destroy) }
+    it { expect(subject.send(:admin?)).to be false }
```

Comment:
I'm not sure you need the `[` and `]`, since you're `*` the parameter, you also don't need to `.to_a` the roles in the `.in?`.

Correct re array evaluation.